src/Widgets/Proxy/ProxyConfigurationPage.vala

```diff
@@ -212,30 +209,34 @@ namespace Network.Widgets {
 
         private void verify_applicable () {
             if (auto_button.active) {
-                apply_button.sensitive = auto_entry.text.strip () != "";
+                enable_sw->sensitive = auto_entry.text.strip () != "";
             } else {
-                apply_button.sensitive = http_entry.text.strip () != "" ||
+                enable_sw->sensitive = http_entry.text.strip () != "" ||
                                         https_entry.text.strip () != "" ||
                                         ftp_entry.text.strip () != "" ||
                                         socks_entry.text.strip () != "";
             }
         }
 
-        private void apply_settings () {
-            if (auto_button.active) {
-                proxy_settings.autoconfig_url = auto_entry.text;
-                proxy_settings.mode = "auto";
-
+        public void apply_settings () {
+            if (enable_sw->active) {
+                if (auto_button.active) {
+                    proxy_settings.autoconfig_url = auto_entry.text;
+                    proxy_settings.mode = "auto";
+
+                } else {
+                    http_settings.host = http_entry.text;
+                    http_settings.port = (int)http_spin.value;
+                    https_settings.host = https_entry.text;
+                    https_settings.port = (int)https_spin.value;
+                    ftp_settings.host = ftp_entry.text;
+                    ftp_settings.port = (int)ftp_spin.value;
+                    socks_settings.host = socks_entry.text;
+                    socks_settings.port = (int)socks_spin.value;
+                    proxy_settings.mode = "manual";
+                }
             } else {
-                http_settings.host = http_entry.text;
-                http_settings.port = (int)http_spin.value;
-                https_settings.host = https_entry.text;
-                https_settings.port = (int)https_spin.value;
-                ftp_settings.host = ftp_entry.text;
-                ftp_settings.port = (int)ftp_spin.value;
-                socks_settings.host = socks_entry.text;
-                socks_settings.port = (int)socks_spin.value;
-                proxy_settings.mode = "manual";
+              proxy_settings.mode = "none";
```

Comment:
The indentation is wrong here. Please correct this to be in line with those lines that were removed.